### PR TITLE
Add null and localhost to allowed origins

### PR DESCRIPTION
**Pull request by karelbilek**

### CODE REVIEW

File: server/https.go [hunk 1]
```diff
@@ -63,7 +63,11 @@ func New(bus *usb.USB) (*server, error) {
 	r.HandleFunc("/call/{session}", s.Call)
 
 	headers := handlers.AllowedHeaders([]string{"Content-Type"})
-	origins := handlers.AllowedOrigins([]string{"https://wallet.trezor.io", "https://dev.trezor.io"})
+
+	// `localhost:8000` is added for easing the wallet development
+	// `null` is for electron apps or chrome extensions
+	origins := handlers.AllowedOrigins([]string{"https://*.trezor.io", "https://localhost:8000", "null"})
```
Comment (jpochyla) [inline server/https.go]: Are you sure `https://*.trezor.io` works?

https://github.com/gorilla/handlers/blob/master/cors.go#L311

Comment (karelbilek) [reply server/https.go]: Hm. It worked when I tried it. But it should not be possible according to this :) what is going on?

Comment (karelbilek) [reply server/https.go]: Oh, it indeed doesn't work, there is some error in mytrezor that doesn't detect the CORS error and acts like everything is fine, just doesn't list any devices.

**Comment by karelbilek** [comment]
I have added custom CORS origin validator. It seems to work fine.

**Pull request merged**